gen3-client/jwt/functions.go

```diff
@@ -164,16 +161,15 @@ func (f *Functions) DoRequestWithSignedHeader(profile string, configFileType str
 		if resp != nil && resp.StatusCode == 401 {
 			isExpiredToken = true
 		} else if err != nil {
```

Comment:
probably can get rid of this `else if` ? Contents in `else if` and `else` look identical

Reply:
Removed in the commit [835c73d](https://github.com/uc-cdis/cdis-data-client/pull/49/commits/835c73dad2f8d5e8d6754be9364117b0d864a35f). Thanks!